### PR TITLE
Add OpenAPI as data source

### PR DESCRIPTION
**Pull request by ryandoowan**

### CODE REVIEW

File: _data/tools.yml [hunk 1]
```diff
@@ -756,6 +756,22 @@
       url: https://www.magnitude.com/drivers/trino-odbc-jdbc
     - urltext: Starburst ODBC driver
       url: https://docs.starburst.io/data-consumer/clients/odbc.html
+- name: OpenAI
+  anchor: openai
+  category: data-source
+  logo: /assets/images/logos/openai.png
+  logosmall: /assets/images/logos/openai-small.png
+  description: |
+    OpenAI is an AI research and deployment company. Their mission is to ensure
```
Comment (nineinchnick) [inline _data/tools.yml:765]: This is something completely different that OpenAPI

**Review by nineinchnick — CHANGES_REQUESTED** [review]

File: _data/tools.yml [hunk 2]
```diff
@@ -756,6 +756,23 @@
...
+  category: data-source
+  logo: /assets/images/logos/openapi.png
+  logosmall: /assets/images/logos/openapi-small.png
+  description: |
+    The OpenAPI Specification is a specification language for HTTP APIs that
+    provides a standardized means to define your API to others. You can quickly
+    discover how an API works, configure infrastructure, generate client code,
+    and create test cases for your APIs.
+
+    Trino has a OpenAPI connector for accessing APIs with an OpenAPI spec.
```
Comment (mosabua) [inline _data/tools.yml:770]: Use any REST API that publishes an OpenAPI specification as a data source in Trino by configuring a catalog with the OpenAPI connector.

**Review by mosabua — APPROVED** [review]
Looks good to me. Can you confirm @nineinchnick ?

File: _data/tools.yml [hunk 3]
```diff
@@ -756,6 +756,24 @@
...
       url: https://docs.starburst.io/data-consumer/clients/odbc.html
+- name: OpenAPI
+  anchor: openapi
+  category: data-source
+  logo: /assets/images/logos/openapi.png
+  logosmall: /assets/images/logos/openapi-small.png
+  description: |
+    The OpenAPI Specification is a specification language for HTTP APIs that
+    provides a standardized means to define your API to others. You can quickly
+    discover how an API works, configure infrastructure, generate client code,
```
Comment (nineinchnick) [inline _data/tools.yml:767]: The point of using this connector is to avoid having to generate a client and use sql instead. I'd remove this phrase

**Comment by mosabua** [comment]
Comments addressed. Looks good now. Shipping.

**Pull request merged**